### DISCUSSION TITLE
[@type/tabulator-tables] issue with extending modules for Formatter/Edittor

### DISCUSSION THREAD

**Discussion by huyit88** [DefinitelyTyped/DefinitelyTyped#67959] [Issues with a @types package] [answered]
Could you please help to see extending module of tabulator?

https://tabulator.info/docs/5.5/modules

`

Tabulator.extendModule("format", "formatters", {
    bold:function(cell, formatterParams){
        return "<strong>" + cell.getValue() + "</strong>"; //make the contents of the cell bold
    },
    uppercase:function(cell, formatterParams){
        return cell.getValue().toUpperCase(); //make the contents of the cell uppercase
    }
}); 

`

In case i add two new formatters bold and uppercase, it seem current Formatter can not handle it. 

It will show error if user try using extending formatter: { title:'', formatter: 'bold'}

[a comment by an automated account is not shown]

**Comment by Martino-Martinez** [comment]
Please change your title to @types/tabulator-tables (the "s" in types is missing). Please also provide more details. I might be able to help.
